routes/web.php

```diff
@@ -11,6 +11,24 @@
 |
 */
 
+    // 获取所有问题列表
+    $router->get('/news/get-all-news',['uses'=>'NewsController@getAllNews']);
+
+    // 显示今天的新闻新增内容
+    $router->get('/news/show-today-news',['uses'=>'NewsController@showTodayNews']);
+
+    // 置顶新闻
+    $router->get('/news/sticky-news',['uses'=>'NewsController@stickyNews']);
```

Comment:
```suggestion
    $router->post('/news/pin-news',['uses'=>'NewsController@pinNews']);
```
改成 `post` 方法，然后把你的方法名改成 `pinNews`。记得改控制器的方法名